src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Marshalling/AttributedMarshallingModelGeneratorFactory.cs

```diff
@@ -278,20 +278,20 @@ private ResolvedGenerator CreateCustomNativeTypeMarshaller(TypePositionInfo info
 
                 FreeStrategy freeStrategy = GetFreeStrategy(info, context);
 
-                if (freeStrategy == FreeStrategy.FreeOriginal)
-                {
-                    marshallingStrategy = new UnmanagedToManagedOwnershipTrackingStrategy(marshallingStrategy);
-                }
+                //if (freeStrategy == FreeStrategy.FreeOriginal)
+                //{
+                //    marshallingStrategy = new UnmanagedToManagedOwnershipTrackingStrategy(marshallingStrategy);
+                //}
```

Comment:
I'll stop commenting, but undescribed commented out code is a red-flag in general.